pkg/common-controller/snapshot_metrics.go

```diff
@@ -0,0 +1,37 @@
+package common_controller
+
+const (
+	createSnapshotOperationName = "CreateSnapshot"
+	snapshottingOperationName   = "Snapshotting"
```

Comment:
What is the difference between snapshotting and createsnapshot?

Reply:
CreateSnapshot is full end to end from VolumeSnapshot CR noticed until it's marked ready.
Snapshotting is defined as when the VolumeSnapshot CR is noticed until a VolumeSnapshotContent is created.

From the design doc:

Snapshotting:
* Start_time: controller notices the first time that there is a new VolumeSnapshot CR to dynamically provision a snapshot
* End_time: controller notices that the CR has a status with CreationTime field set to be non-nil

CreateSnapshot
* Start_time: controller notices the first time that there is a new VolumeSnapshot CR(both dynamic and pre-provisioned cases)
* End_time: controller notices that the CR has a status with Ready field set to be true

@yuxiangqian is it worth only having CreateSnapshot and DeleteSnapshot operations? What sort of use cases are there for needing the snapshotting operation?

Reply:
My main ask here is to make the names more obvious what they're measuring. What about something like: "CreateSnapshot" and "CreateSnapshotAndReady"?

Reply:
CreateSnapshotAndReady sounds good, that is more clear to me. @yuxiangqian does this sound good? 

In this case, we would actually rename Snapshotting to CreateSnapshot, and rename CreateSnapshot to CreateSnapshotAndReady:

CreateSnapshot:

* Start_time: controller notices the first time that there is a new VolumeSnapshot CR to dynamically provision a snapshot
* End_time: controller notices that the CR has a status with CreationTime field set to be non-nil

CreateSnapshotAndReady
* Start_time: controller notices the first time that there is a new VolumeSnapshot CR(both dynamic and pre-provisioned cases)
* End_time: controller notices that the CR has a status with Ready field set to be true

Reply:
Sounds good to me.

Reply:
Changed the metric types to reflect the above.